app/build.gradle

```diff
@@ -67,11 +67,9 @@ android {
 }
 
 dependencies {
-    implementation 'com.android.support.constraint:constraint-layout:1.1.3'
```

Comment:
I'm not sure why this is in the diff since this was merged in via a previous PR. Sorry about this noise.